docs/connect/clients/java.md

```diff
@@ -187,104 +187,126 @@ public class Main {
 }
 ```
 
-### Example: Indexing and querying JSON documents
+### Production usage
 
-Make sure that you have Redis Stack and `Jedis` installed. 
+### Configuring Connection pool
+As was covered in the previous section, it's recommended to use `JedisPool` or `JedisPooled`. 
+`JedisPooled` is available since Jedis 4.0.0 and is a wrapper around `JedisPool` that provides a simpler API.

```

Comment:
`wrapper` is wrong term here.
It allows similar (exactly same) connection (resource) maintenance as JedisPool.
But, yes, it provides simpler API.

Reply:
Fixed. Thanks!